### DISCUSSION TITLE
Decoupled CMS with GrapesJs

### DISCUSSION THREAD

**Discussion by hannydevelop** [GrapesJS/grapesjs#5183] [Show and tell]
Myself and some friends are building an open source "Decoupled Visual CMS"  with GrapesJs. We're adding finishing touches now and hope to move it to Github early next month. 

This tool allows you to visually build web applications using languages and tools you already use. For instance, you can use GrapesJs' editor to visually build web pages, add workflows and export them into Vue/NodeJs stack without writing any code.

I'll share the repository once it's ready so everyone can take a look.

**Comment by anshmarelab85** [comment]
hi @hannydevelop ,
how are you,
I want to see your product can you please provide me git repo
